kroxylicious-integration-test-support/src/main/java/io/kroxylicious/test/tester/KroxyliciousConfigUtils.java

```diff
@@ -27,6 +29,8 @@ private KroxyliciousConfigUtils() {
     }
 
     public static final String DEFAULT_VIRTUAL_CLUSTER = "demo";
+    public static final String DEFAULT_LISTENER_NAME = "default";
```

Comment:
It's a pain that this testing detail percolates into so many tests, have refactored to introduce some methods for obtaining default listener builders.

https://github.com/k-wall/kroxylicious/pull/52